packages/esm-patient-chart-app/src/visit/hooks/useVisitAttributeType.tsx

```diff
@@ -35,21 +35,11 @@ const visitAttributeTypeCustomRepresentation =
   'custom:(uuid,display,name,description,datatypeClassname,datatypeConfig)';
 
 export function useVisitAttributeType(uuid) {
-  const { data, error } = useSWRImmutable<FetchResponse<VisitAttributeType>, Error>(
+  const { data, error } = useSWRImmutable<FetchResponse<VisitAttributeType>, any>(
```

Comment:
Why the change to `any`?

Reply:
Resolved, thanks @ibacher for the catch!